### ISSUE TITLE
Plugin broken: ImportError: No module named qgissettingmanager

### ISSUE THREAD

**Issue by pcav** [Septima/qgis-qlrbrowser#57]
The just upgraded plugin is unusable.

**Comment by Frederikssund** [comment]
#### The full error message:

```
Kunne ikke indlæse plugin QlrBrowser pga. en fejl ved kald af dens classFactory() metode
Traceback (most recent call last):
  File "c:/OSGeo4w/apps/qgis/./python\qgis\utils.py", line 219, in startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "C:/Users/bvtho/.qgis2/python/plugins\QlrBrowser\__init__.py", line 34, in classFactory
    from .qlrbrowser import QlrBrowser
  File "c:/OSGeo4w/apps/qgis/./python\qgis\utils.py", line 478, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/bvtho/.qgis2/python/plugins\QlrBrowser\qlrbrowser.py", line 32, in 
    from .ui.dockwidget import DockWidget
  File "c:/OSGeo4w/apps/qgis/./python\qgis\utils.py", line 478, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/bvtho/.qgis2/python/plugins\QlrBrowser\ui\dockwidget.py", line 28, in 
    from ..core.filesystemmodel import FileSystemModel, FileSystemRecursionException
  File "c:/OSGeo4w/apps/qgis/./python\qgis\utils.py", line 478, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/bvtho/.qgis2/python/plugins\QlrBrowser\core\filesystemmodel.py", line 6, in 
    from qlrbrowser_settings import QlrBrowserSettings
  File "c:/OSGeo4w/apps/qgis/./python\qgis\utils.py", line 478, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:/Users/bvtho/.qgis2/python/plugins\QlrBrowser\core\qlrbrowser_settings.py", line 25, in 
    from ..qgissettingmanager import SettingManager
  File "c:/OSGeo4w/apps/qgis/./python\qgis\utils.py", line 478, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)

ImportError: No module named qgissettingmanager

Python-version:
2.7.5 (default, May 15 2013, 22:44:16) [MSC v.1500 64 bit (AMD64)]

QGIS-version:
2.8.2-Wien Wien, 1b929ef
```

Installation : I've extracted the relevant directory from the sourcecode zipfile and copied it to the QGIS plugin directory.
Versions:  Windows 7/64 bit, QGIS 2.8.2 64 bit

**Comment by Frederikssund** [comment]
The obvious error is that the "qgissettingmanager" subdirectory in the "QlrBrowser" directory is empty.

**Comment by AsgerPetersen** [comment]
It is caused by us switching qgissettingsmanager from a regular sub directory to a Git submodule. Submodules apparently dont get checked out with `git archive`. And I didnt think of this as a potential problem.

In our tests we just copy the new version on top of the old. This method obviously doesn't catch files disappearing.

We have a large improvement potential :-)

**Comment by Frederikssund** [comment]
If you take the contents of "Septima\qgisettingmanager" and copy it to the empty directory the qlrbrowser plugin works. And the day you don't improve, you're dead !!  ;-)

**Comment by AsgerPetersen** [comment]
This should be fixed in version 0.2.5 which I have just uploaded for approval.

At the same time I have disapproved version 0.2.4, so users dont get this version anymore.

@Frederikssund I prefer to have automated build and packaging. Copying files around manually will eventually lead to broken packages *cough*

**Comment by Frederikssund** [comment]
Yeah, right, tell me about it ;-) 
(It's known a "the DOS attitude": It's GOTTA WORK !! - somehow)
